service/history/visibilityQueueTaskExecutor.go

```diff
@@ -381,7 +381,12 @@ func (t *visibilityQueueTaskExecutor) processCloseExecution(
 	if err != nil {
 		return err
 	}
-	return t.cleanupExecutionInfo(ctx, task)
+
+	config := t.shard.GetConfig()
```

Comment:
visibilityQueueTaskExecutor should have a field of dynamicconfig.BoolPropertyFnWithNamespaceIDFilter.

Reply:
Why not get config like this?